### PR TITLE
Fix/disable no magic number rule

### PR DESCRIPTION
**Pull request by KarishmaBothara**
PR - Fix or disables "no magic number" rule..
Some places it is just checking for 0.. Have disabled the check their over creating a variable.

### CODE REVIEW

**Comment by benxgao** [comment]
Now that we had prevented linting errors from `yarn lint`, how about we pause turning on them one by one for now.

We could have a session to discuss within the team to decide which rules we should keep and which to remove. After that, we could split any further relevant refatoring tasks within the team.

Then we could close this ticket https://centralitydev.atlassian.net/jira/software/projects/IND/boards/142?selectedIssue=IND-239 , and open a new one for further discussion. What do you think? cc @alexsednz @holygits

File: packages/api/src/derives/fees/estimateFee.ts [hunk 1]
```diff
@@ -29,11 +29,12 @@ import {catchError, first, map, switchMap} from 'rxjs/operators';
 export function estimateFee(api: ApiInterfaceRx) {
   // We generate fake signature data here to ensure the estimated fee will correctly match the fee paid when the extrinsic is signed by a user.
   // This is because fees are currently based on the byte length of the extrinsic
+  const genesisBlock = 0;
```
Comment (holygits) [inline packages/api/src/derives/fees/estimateFee.ts:32]: `genesisBlockNumber`

File: packages/types/src/runtime/sylo/index.ts [hunk 2]
```diff
@@ -85,8 +86,9 @@ export class PendingInvite extends Struct {
   }
 
   toJSON() {
+    const bitLenght = -1;
```
Comment (holygits) [inline packages/types/src/runtime/sylo/index.ts:89]: `bitLength`

**Review by holygits — COMMENTED** [review]
This lint might be better as a warning, especially don't like we need to annotate lot's of obvious zero checks

**Comment by KarishmaBothara** [comment]
> This lint might be better as a warning, especially don't like that we'd need to annotate lots of obvious zero checks

In that case.. shall I decline this PR? and just turn it off to warning.

**Comment by KarishmaBothara** [comment]
Not required. Planning to turn this rule to warning

**Pull request closed without merging**